src/main/java/org/broadinstitute/hellbender/cmdline/ReadFilterArgumentDefinitions.java

```diff
@@ -48,4 +48,9 @@ private ReadFilterArgumentDefinitions(){}
     public static final String SAMPLE_NAME = "sample";
 
     public static final String KEEP_INTERVAL_NAME = "keep-intervals";
+
+    public static final String SOFT_CLIPPED_RATIO_THRESHOLD = "soft-clipped-ratio-threshold";
+    public static final String SOFT_CLIPPED_LEADING_TRAILING_RATIO_THRESHOLD = "soft-clipped-leading-trailing-ratio";
+
+    public static final String INVERT_FILTER = "invert-filter";
```

Comment:
I would make this "invert-soft-clip-ratio" or something like that to make it a little clearer that its associated with THIS filter not just any filter on the command line

Reply:
Sure.  Really, this should be a feature of every reader.  I'll add a ticket for it.